src/dune_rules/install_rules.ml

```diff
@@ -1047,7 +1047,7 @@ let package_deps (pkg : Package.t) files =
         let* res = Dune_engine.Build_system.execute_rule rule in
         loop_files
           rules_seen
-          (Dep.Facts.paths ~expand_aliases:true res.deps
+          (Dep.Facts.paths ~expand_aliases:true res.facts
```

Comment:
@rgrinberg Are you sure passing `~expand_aliases:true` here is correct? Shouldn't we be passing `Execution_parameters.expand_aliases_in_sandbox` instead of expanding aliases unconditionally?

Reply:
These aren't user rules - so dune has the freedom to pick the correct behavior. I think `expand_aliases:true` is correct, since we are using an alias to collect all the dependency of a particular package.

Reply:
But... wouldn't a user expect changing this configuration setting affect what install rules are doing?

Reply:
The install rules aren't really configurable in this way. E.g. when one writes:

```
(rule
 (deps (package foo))
 (..))
```

The expectation is that the package `foo` is built *and* its artifacts are present in _build/install. There's an alias that introduces the layer of indirection between these two facts, and for it to work as expected, `expand_aliases` has to be set. Now we could get rid of using the alias this way, but it we'll need to reimplement the install rules and I recall running into problems doing it without the level of indirection from the alias.

Reply:
I see, thanks! I was mostly wondering if we can avoid calling `Dep.Facts.paths` in two different ways. It sounds like we can't.